std/experimental/allocator/building_blocks/aligned_block_list.d

```diff
@@ -130,7 +130,7 @@ public:
     static if (stateSize!ParentAllocator) ParentAllocator parent;
     else alias parent = ParentAllocator.instance;
 
-    enum ulong alignment = theAlignment;
+    enum ulong alignment = ParentAllocator.alignment;
```

Comment:
If `alignment` means the alignment of the memory returned by `allocate`, it would appear this should be `Allocator.alignment` instead of `ParentAllocator.alignment`.

Also maybe it should have a guard like:
```d
static if (hasStaticallyKnownAlignment!Allocator)
    enum alignment = Allocator.alignment;
```
The existence of `std.experimental.allocator.common : hasStaticallyKnownAlignment` implies that not all allocators have statically known alignments, but this may be obsolete or not applicable to the specific allocators required here.